src/components/d-active-card/DActiveCard.tsx

```diff
@@ -0,0 +1,68 @@
+// libraries
+import React from 'react'
+import {ImageBackground, Text, TouchableOpacity, View } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient';
+
+// Styles
+import {styles} from './dActiveCard-styles';
+
+// interface
+import { Challenge } from '../../screens/d-active-challenges/DActiveChallenges';
+
+// theme
+import { SPACING } from '../../theme/spacing';
+import { BACKGROUNDGRADIENTS } from '../../theme/color';
+
+// constants
+import { buttonsData,outerRing,focus,scan,square,follow,noChallenge } from '../../constants/dActivechallenges-constants';
+import { ResizeMode } from '../../constants/common-constants';
+
+/** DActiveCard: {This function shows active cards according to the data given.} */
+const DActiveCard = ({data}:{data:Challenge}) => {
+    let imageName = noChallenge;
+    /** getImage: {This function returns the background image according to the name of challenge.} */
+    const getImage=(name:string)=>{
+        switch (name) {
+            case "Outer Ring":
+                imageName = outerRing;
+                break;
+            case "Focus":
+                imageName = focus;
+                break;
+            case "Scan":
+                imageName = scan;
+                break;
+            case "Square":
+                imageName = square;
+                break;
+            case "Follow":
+                imageName = follow;
+                break;
+        }
+        return imageName;
+    }
+  return (
+    <View style={styles.container}>
+        <ImageBackground source={getImage(data.name)} resizeMode={ResizeMode.Contain} style={[styles.background,{elevation:data.dropShadow?data.dropShadow:0},{elevation:data.cardShadow?data.cardShadow:0}]}>
```

Comment:
why style is like this

Reply:
Because I am handling it dynamically according to the json data and not all the cards have shadow so for handling it I used inline style.